### DISCUSSION TITLE
Add UI to use options for storing the SDK's

### DISCUSSION THREAD

**Discussion by stayallive** [stayallive/wp-sentry#111] [Ideas]
Because on WordPress.com you cannot edit the `wp-config.php` we should make it possible to have the DSN loaded from the WordPress options API using an admin settings UI.

However, this options API loading should be disabled (and not registered to prevent extra database load) when the `wp-config.php` key has a DSN set and should either hide or disable the admin settings UI too.

Support question sparking this request:
> https://wordpress.org/support/topic/using-on-site-hosted-on-wordpress/

**Comment by MadCap-Software-Marketing** [comment]
This seems like exactly what I need. Is it on the road map? Or, is there any other solution to get wp-sentry working on a Wordpress.com hosted site?

**Comment by MadCap-Software-Marketing** [comment]
I was able to find a workaround. It is possible to update the wp-config.com on a WordPress hosted site via SFTP. Such as shown here:
https://developer.wordpress.com/docs/developer-tools/sftp/

**Comment by stayallive** [comment]
Ah, didn't know this was a WP.com feature. Okay then this has even lower priority then I thought 😎

Ah the catch is that you need to be on the Creator or Entrepreneur plan, so not a solution for everyone.
